spyder/py3compat.py

```diff
@@ -82,6 +82,16 @@
 #==============================================================================
 # Strings
 #==============================================================================
+def is_type_text_string(obj):
+    """Return True if `obj` is type text string, False if it is anything else,
+    like binary data (Python 3) or QString (Python 2, PyQt API #1)"""
+    if PY2:
+        # Python 2
+        return type(obj) in [basestring]

```

Comment:
Why do you need to enclose `basetring` in brackets?

Reply:
I will change this :+1: